src/kvstore/RocksEngineConfig.cpp

```diff
@@ -42,6 +42,22 @@ DEFINE_int32(rocksdb_batch_size,
 DEFINE_string(part_man_type,
               "memory",
               "memory, meta");
+
+static bool validateRocksdbStatsLevel(const char* flagname, int value) {
+    if (value < rocksdb::StatsLevel::kAll && value >= 0) {
+        return true;
+    }
+
+    VLOG(3) << "Invalid value for --" << flagname << ": " << value;
+    return false;
+}
+
+DEFINE_int32(rocksdb_stats_level, 0,
+             "Statistics Level for RocksDB. Default is 0 (kExceptHistogramOrTimers)");
+DEFINE_validator(rocksdb_stats_level, &validateRocksdbStatsLevel);
+
+DEFINE_int32(rocksdb_stats_dump_period_sec, 0, "DBOptions::stats_dump_period_sec for RocksDB");
```

Comment:
I remember this option only control the interval for dumping stats into the LOG file, and if we want to get the stats in string, need to call `db->GetProperty()` to get the string version.